### PR TITLE
Ensuring consistency in FEniCS scripts for FSI tutorial cases 

### PR DESCRIPTION
**Pull request by IshaanDesai**
This PR introduces minor changes to FEniCS scripts for perpendicular flap cases coupled with OpenFOAM and SU2, to ensure consistency.

### CODE REVIEW

File: FSI/flap_perp_2D/OpenFOAM-FEniCS/Allclean [hunk 1]
```diff
@@ -3,9 +3,6 @@ cd ${0%/*} || exit 1    # Run from this directory
 
 echo "Cleaning..."
 
-# Source tutorial clean functions
-. $WM_PROJECT_DIR/bin/tools/CleanFunctions
-
```
Comment (IshaanDesai) [inline FSI/flap_perp_2D/OpenFOAM-FEniCS/Allclean:8]: @MakisH is it okay to remove this? On my local machine no such directory of file exists and hence the `Allclean` script halts here and no cleaning is done.

Comment (MakisH) [reply FSI/flap_perp_2D/OpenFOAM-FEniCS/Allclean:8]: No, this comes from OpenFOAM and it is necessary to run the OpenFOAM clean scripts. It should exist when you have OpenFOAM loaded.

Comment (IshaanDesai) [reply FSI/flap_perp_2D/OpenFOAM-FEniCS/Allclean:8]: Okay got it. The habit of running `./Allclean` in the terminal which runs FEniCS led to believe something was going wrong here. I will revert this change now :+1:

**Review by BenjaminRodenberg — APPROVED** [review]
Code looks good to me. Didn't run the case, though.

**Pull request merged**